File: src/main/java/um/tesoreria/facturador/model/dto/ChequeraFacturacionElectronicaDto.java
package um.tesoreria.facturador.model.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import um.tesoreria.facturador.tool.Jsonifier;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ChequeraFacturacionElectronicaDto {

    private Long chequeraFacturacionElectronicaId;
    private Long chequeraId;
    private String cuit = "";
    private String razonSocial = "";
    private String domicilio = "";
    private String email = "";
    private String condicionIva = "";

    public String jsonify() {
        return Jsonifier.builder(this).build();
    }

}

File: src/main/java/ar/edu/um/tesoreria/facturador/kotlin/model/dto/ChequeraFacturacionElectronicaDto.kt
package ar.edu.um.tesoreria.facturador.kotlin.model.dto

data class ChequeraFacturacionElectronicaDto(

    var chequeraFacturacionElectronicaId: Long? = null,
    var chequeraId: Long? = null,
    var cuit: String = "",
    var razonSocial: String = "",
    var domicilio: String = "",
    var email: String = "",
    var condicionIva: String = ""

)
